R/rbm.R

```diff
@@ -235,7 +255,14 @@ predict.rbm <- function (object, newdata, type='probs', omit_bias=TRUE, ...) {
     } else if(attr(class(newdata), 'package') != 'Matrix'){
       stop('Unsupported class for rmb: ', paste(class(newdata), collapse=', '))
     }
-    
+    #Scale if scaled during training
+    if (!is.null(object$scaled)) {
+      for (i in 1:length(object$scaled$scaled)){
+        if (scaled$scaled[i] == 1){
```

Comment:
Typo here: `scaled$scaled` should be `object$scaled$scaled`.

I fixed this in the master branch, but I also need to write unit tests for this package, lol.